docs/configuration.rst

```diff
@@ -204,6 +204,28 @@ or:
 
 .. note:: However, this is mutually exclusive with the ``ignore`` key.
 
+You may also specify inline ignore paths as a list of strings:
+
+.. code-block:: yaml
+
+ # For all rules
+ ignore:
+   - '*.dont-lint-me.yaml'
+   - '/bin/'
+   - '!/bin/*.lint-me-anyway.yaml'
+
+ extends: default
+
+ rules:
+   key-duplicates:
+     ignore:
+       - 'generated'
+       - '*.template.yaml'
+   trailing-spaces:
+     ignore:
+       - '*.ignore-trailing-spaces.yaml'
+       - 'ascii-art/*'
+
```

Comment:
I fear it won't help readability to add a new paragraph. Instead, what do you think about changing the first sentence to:
```diff
 It is possible to exclude specific files or directories, so that the linter
-doesn't process them.
+doesn't process them. They can be provided either as a list of paths, or as a
+bulk string.
```
and give simply one example if the next block:
```yaml
 ignore: |
   /this/specific/file.yaml
   all/this/directory/
   *.template.yaml

 # or:

 ignore:
   - /this/specific/file.yaml
   - all/this/directory/
   - *.template.yaml
```
?